src/main/java/com/hubspot/jackson/datatype/protobuf/ProtobufEnumAsNumber.java

```diff
@@ -0,0 +1,79 @@
+package com.hubspot.jackson.datatype.protobuf;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.BeanProperty;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.deser.ContextualDeserializer;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import com.google.common.base.Preconditions;
+import com.google.protobuf.ProtocolMessageEnum;
+
+public final class ProtobufEnumAsNumber {
```

Comment:
I'm happy to reorganize this into two separate classes if you prefer that style.